pyaedt/icepak.py

```diff
@@ -3430,16 +3462,35 @@ def assign_stationary_wall(
         props["Thickness"] = (thickness,)
         props["Solid Material"] = material
         props["External Condition"] = boundary_condition
-        props["Heat Flux"] = heat_flux
-        props["Temperature"] = temperature
-        if htc_dataset is None:
-            props["Heat Transfer Coefficient"] = htc
-        else:
-            props["Heat Transfer Coefficient Variation Data"] = {
-                "Variation Type": "Temp Dep",
-                "Variation Function": "Piecewise Linear",
-                "Variation Value": '["1w_per_m2kel", "pwl({},Temp)"]'.format(htc_dataset),
-            }
+        if "htc_dataset" in kwargs:  # backward compatibility
+            warnings.warn("``htc_dataset`` argument is being deprecated. Please create a dictionary as per"
```

Comment:
```suggestion
            warnings.warn("``htc_dataset`` argument is being deprecated. Ceate a dictionary as per"
```
Omit "please" in technical documentation.